.github/workflows/testing.yaml

```
@@ -40,6 +40,61 @@ jobs:
+        run: |
+          cd my-next-app
+          cat <<EOT > ./auth.ts
+          import NextAuth from 'next-auth';
+          import Google from "next-auth/providers/google"
+
+          export const { auth, handlers, signIn, signOut } = NextAuth({
+            providers: [Google],
+          });
+          EOT
+
+      - name: Create a footer.tsx component:
```

actionlint

could not parse as YAML: yaml: line 60: mapping values are not allowed in this context

(syntax-check)